<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout"/>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>600</width>
     <height>24</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_File">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="actionNew_File"/>
    <addaction name="action_Open_File"/>
    <addaction name="action_Close_File"/>
    <addaction name="action_Save"/>
    <addaction name="separator"/>
    <addaction name="action_Quit"/>
   </widget>
   <widget class="QMenu" name="menu_Actions">
    <property name="title">
     <string>&amp;Actions</string>
    </property>
    <addaction name="actionAdd_Node"/>
    <addaction name="actionAdd_Subnode"/>
    <addaction name="separator"/>
    <addaction name="action_Remove_Node"/>
    <addaction name="separator"/>
    <addaction name="actionAdd_Attribute"/>
    <addaction name="separator"/>
    <addaction name="actionFind"/>
   </widget>
   <widget class="QMenu" name="menuAbout">
    <property name="title">
     <string>About</string>
    </property>
    <addaction name="actionAbout_QXmlGraphicEditor"/>
    <addaction name="actionAbout_Qt"/>
   </widget>
   <addaction name="menu_File"/>
   <addaction name="menu_Actions"/>
   <addaction name="menuAbout"/>
  </widget>
  <action name="action_Open_File">
   <property name="text">
    <string>&amp;Open File</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="action_Close_File">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>&amp;Close File</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+W</string>
   </property>
  </action>
  <action name="action_Save">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>&amp;Save</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="action_Quit">
   <property name="text">
    <string>&amp;Quit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actionAdd_Node">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Add Node</string>
   </property>
  </action>
  <action name="actionAdd_Subnode">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Add Subnode</string>
   </property>
  </action>
  <action name="action_Remove_Node">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Remove Node</string>
   </property>
  </action>
  <action name="actionAdd_Attribute">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Node Attributes</string>
   </property>
  </action>
  <action name="actionFind">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Find</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+F</string>
   </property>
  </action>
  <action name="actionAbout_QXmlGraphicEditor">
   <property name="text">
    <string>About QXmlGraphicEditor</string>
   </property>
  </action>
  <action name="actionAbout_Qt">
   <property name="text">
    <string>About Qt</string>
   </property>
  </action>
  <action name="actionNew_File">
   <property name="text">
    <string>New File</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>action_Quit</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>299</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
